### DISCUSSION TITLE
Extracted merged sub from video as .srt file or convert to .srt?

### DISCUSSION THREAD

**Discussion by rieje** [mifi/lossless-cut#2172] [Q&A] [answered]
I merged .srt file into video file, then trim it (normal mode, not smart cut which unfortunately messes up timing of subtitles). 

After trimming the video, I would like to edit the subtitles. In LosslessCut, after "Extract this track as file", I'm not sure what type of file is produced--when I open the file, I see some subtitle text but also a lot of gibberish so thinking this is a binary file. Is it possible for LosslessCut to extract as an .srt file or to convert this binary file as an .srt file so that I can modify both the subtitles and timestamps then merge it back into the video?

**Comment by mifi** [comment, marked as answer]
1. open your file
2. open "Tracks"
3. Disable all tracks except the subtitle track
4. select output format as `srt`
5. Export
6. you should now have an .srt file
